frontend/src/contrainers/requestContainer/requestContainer.js

```diff
@@ -0,0 +1,43 @@
+import React, { useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Request from '../../components/request/request';
+import { withRouter } from 'react-router-dom';
+import { requestCarpool } from '../../modules/carpoolRequest';
+
+RequestContainer.propTypes = {};
+
+function RequestContainer({ history }) {
+  const dispatch = useDispatch();
+  const user = useSelector(user => ({
+    user: user.user,
+  }));
+
+  //TODO: it should be loaded from DB
+  const fromList = ['SNU station', 'Nakseongdae station', 'Nokdu Street'];
+  const toList = ['301 building', 'Student Center', 'Dormitory three-way'];
+  const minimumPassenger = ['2', '3', '4'];
+
+  const onClickRequest = useCallback(
+    payload => {
+      dispatch(requestCarpool(payload));
+      history.push('/waiting');
+    },
```

Comment:
for clarity please change `payload` => `({ userId, from, to, minimumPassenger })`